parts/language/lang_vi.lua

```diff
@@ -0,0 +1,1070 @@
+        databaseError="Lỗi cơ sở dữ liệu",
+        invalidFormat="Định dạng không hợp lệ",
+        invalidArguments="Đối số không hợp lệ",
+        tooFrequent="Quá thường xuyên",
+        notAvailable="Không khả dụng",
+        noPermission="Không có quyền",
+        roomNotFound="Không tìm thấy phòng",
+
+        -- Controllers
+        WebSocket={
+            invalidConnection="Kết nối không hợp lệ",
+            invalidAction="Hành động không hợp lệ",
+            playerNotFound="Không tìm thấy người chơi",
+            connectionFailed="Kết nối thất bại",
+        },
+        -- Filters
+        CheckPermission={
+            playerNotFound="Không tìm thấy người chơi",
+        },
+        -- Plugins
+        ConnectionManager={
+            playerInvalid="Player invalid", -- ???
+            playerNotFound="Không tìm thấy người chơi",
+            connectionReplaced="Connection replaced",
+        },
+        NoticeManager={
+            noticeNotFound="Notice not found",
+        },
+        PlayerManager={
+            invalidCode="Mã không hợp lệ",
+            invalidEmail="Email không hợp lệ",
+            playerNotFound="Không tìm thấy người chơi",
+            invalidEmailPass="Email hoặc người chơi không hợp lệ",
+            emailExists="Email đã tồn tại",
+            emailSendError="Lỗi gửi email",
+        },
+        -- Strategies
+        PlayerRole={
+            invalidRole="Invalid role",
+            invalidTarget="Invalid target",
+        },
+        PlayerType={
+            invalidType="Invalid type",
+            roomFull="Phòng đã đầy",
+        },
+        RoomJoin={
+            wrongPassword="Sai mật khẩu",
+        },
+    },
+
+    tooFrequent="Request too frequently",
+    roomPasswordChanged="Mật khẩu của phòng đã thay đổi",
+    oldVersion="Phiên bản $1 hiện đã ra mắt",
+    versionNotMatch="Phiên bản không khớp",
+    notFinished="Sắp ra mắt!",
+
+    noUsername="Vui lòng nhập email của bạn",
+    wrongEmail="Địa chỉ email không hợp lệ",
+    wrongCode="Mã xác minh không hợp lệ",
+    noPassword="Vui lòng nhập mật khẩu của bạn",
+    diffPassword="Mật khẩu không khớp",
+    checkEmail="Yêu cầu đăng ký đã được gửi.",
+
+    wsFailed="Kết nối WebSocket đã thất bại: $1",
+    wsClose="WebSocket đã đóng: $1",
+    netTimeout="Kết nối đã quá hạn",
+    serverDown="Ối! Server bị sập rồi",
+    requestFailed="Yêu cầu thất bại",
+
+    onlinePlayerCount="Trực tuyến: $1",
+    createRoomSuccessed="Phòng đã được tạo",
+    playerKicked="$1 removed $2 from room",
+    becomeHost="$1 sẽ là chủ phòng",
+    started="Đang chơi",
+    joinRoom="$1 vừa vào phòng.",
+    leaveRoom="$1 vừa rời phòng.",
+    roomRemoved="Phòng đã bị loại bỏ",
+    ready="Sẵn sàng",
+    spectating="Đang theo dõi",
+
+
+
+    keySettingInstruction="Nhấn một phím để chỉ định phím đó\nescape (esc): Huỷ chỉ định\nbackspace: Xoá phím đã chỉ định",
+    customBGhelp="Chèn một tấm ảnh vào đây để áp dụng ảnh nền tuỳ chỉnh",
```

Comment:
Line 196: I think using ``Chèn`` (insert) can make some players misunderstanding
How about if we use ``Thả`` or ``Ném``?